src/main/webapp/i18n/de/exercise.json

```diff
@@ -75,11 +75,15 @@
             "feedbackHint": "Das Feedback ist ein erläuternder Text an Studierende, um ihr Leistungsniveau besser zu verstehen.",
             "title": "Titel",
             "shortName": "Kurzname",
+            "release": "Veröffentlichung",
             "releaseDate": "Veröffentlichungsdatum",
+            "start": "Start",
             "startDate": "Startdatum",
+            "due": "Frist",
             "dueDate": "Einreichungsfrist",
```

Comment:
Did you ensure that the `releaseDate`, `startDate` and `dueDate` translations are still in use somewhere? Otherwise I think they should be removed

Reply:
Double checked again. Used plenty of times.